test/e2e/main_test.go

```diff
@@ -151,6 +151,12 @@ func TestReportingE2E(t *testing.T) {
 			timeout:       reportTestTimeout,
 			skip:          !runAWSBillingTests,
 		},
+		{
+			name:          "cluster-persistentvolumeclaim-request",
+			queryName:     "cluster-persistentvolumeclaim-request",
+			newReportFunc: testFramework.NewSimpleReport,
+			timeout:       reportTestTimeout + time.Minute,
```

Comment:
I think timeout can just be `reportTestTimeout`.

Reply:
Fixed in commit 1e1ff2d9f183d3c30a279ea8f1865aa378d8c7ff